packages/accordion/README.md

```diff
@@ -25,75 +25,72 @@ When looking to leverage the `Accordion` and `AccordionItem` base class as a typ
 import { Accordion, AccordionItem } from '@spectrum-web-components/accordion';
 ```
 
-## Example
+### Anatomy
 
-```html
-<sp-accordion>
-    <sp-accordion-item label="Heading 1">
-        <div>Item 1</div>
-    </sp-accordion-item>
-    <sp-accordion-item disabled label="Heading 2">
-        <div>Item 2</div>
-    </sp-accordion-item>
-    <sp-accordion-item label="Heading 3">
-        <div>Item 3</div>
-    </sp-accordion-item>
-    <sp-accordion-item label="Heading 4">
-        <div>Item 4</div>
-    </sp-accordion-item>
-    <sp-accordion-item label="Heading 5">
-        <div>Item 5</div>
-    </sp-accordion-item>
-    <sp-accordion-item label="Heading 6">
-        <div>Item 6</div>
-    </sp-accordion-item>
-</sp-accordion>
-```
+The accordion consists of several key parts:
 
-## Allow Multiple
+- A container element that manages the accordion behavior
+- Individual accordion items that can be expanded or collapsed
+- Each item has a header with a label and chevron icon
+- Content that is revealed when an item is expanded
 
 ```html
-<sp-accordion allow-multiple>
-    <sp-accordion-item label="Heading 1">
-        <div>Item 1</div>
-    </sp-accordion-item>
-    <sp-accordion-item disabled label="Heading 2">
-        <div>Item 2</div>
-    </sp-accordion-item>
-    <sp-accordion-item label="Heading 3">
-        <div>Item 3</div>
-    </sp-accordion-item>
-    <sp-accordion-item label="Heading 4">
-        <div>Item 4</div>
-    </sp-accordion-item>
-    <sp-accordion-item label="Heading 5">
-        <div>Item 5</div>
-    </sp-accordion-item>
-    <sp-accordion-item label="Heading 6">
-        <div>Item 6</div>
+<sp-accordion>
```

Comment:
I wanted to make examples that followed our a11y recommendations for meaningful labels that describe content, so I got all meta here.